src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -142,14 +143,14 @@ void tabulate() {
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
     checkConfigVersionMatchesApp(config);
     boolean tabulationSuccess = false;
-    //noinspection ConstantConditions
-    if (config != null && config.validate() && setUpLogging(config)) {
+    if (config.validate() && setUpLogging(config)) {
```

Comment:
Why is the null check unnecessary here? It looks like it's possible for `loadContestConfig` to return a null value.

Reply:
Yes, but checkConfigVersionMatchesApp already dereferences `config`, so it can't possibly be null by the time execution reaches this point.